File: atcoder/abs/00-pracrice/main.ts
/*
 * atcoder typescript template
 */
import * as fs from "fs";

// stdin
const inputs: string = (fs.readFileSync(process.stdin.fd, "utf8"))
const inputArray: string[] = inputs.split(/\s/)
let inputIndex = 0;

// atcoder lib:stdin
function getNext(): string {
	return inputArray[inputIndex++];
}

function getNextNum(): number {
	return Number(getNext());
}

function getNums(N: number): number[] {
	const arr: number[] = [];
	for (let i: number = 0; i < N; ++i) {
		arr[i] = getNextNum();
	}
	return arr;
}

// lib:stdout
let outputBuffer: string = ""
function print(out: string | number | bigint): void;
function print<T>(out: Array<T>, separator: string): void;
function print<T>(out: string | number | bigint | Array<T>, separator?: string) {
	if (Array.isArray(out)) {
		outputBuffer += out.join(separator);
	} else {
		outputBuffer += out;
	}
}

function println(out: string | number | bigint): void;
function println<T>(out: Array<T>, separator: string): void;
function println<T>(out: string | number | bigint | Array<T>, separator?: string) {
	if (Array.isArray(out)) {
		print(out, separator || "");
	} else {
		print(out);
	}
	print("\n");
}

function flush(): void {
	console.log(outputBuffer);
}

// main
function main() {
	// Input
	const [a, b, c] = getNums(3);
	const s: string = getNext();

	// Answer
	const sum = a + b + c

	// Out
	console.log(`${sum} ${s}`);
}

// main
main();
flush();

File: atcoder/abs/00-pracrice/main.go
// Copyright (c) 2023 Furukawa, Atsushi <atsushifx@aglabo.com>
//
// This software is released under the MIT License.
// https://opensource.org/licenses/MIT

package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

// Global Variables
const IOBUFFSIZE = 10 ^ 6

var Stdin = bufio.NewScanner(os.Stdin)
var Stdout = bufio.NewWriter(os.Stdout)

// atcoder lib
func getNext() string {
	Stdin.Scan()
	return Stdin.Text()
}

func getNextInt() int {
	var i int
	var e error
	i, e = strconv.Atoi(getNext())
	if e != nil {
		panic(e)
	}
	return i
}

// initialize
func initialize() {
	stdinBuffer := make([]byte, IOBUFFSIZE)
	Stdin.Buffer(stdinBuffer, IOBUFFSIZE)
	Stdin.Split(bufio.ScanWords)
}

// main
func main() {
	initialize()
	defer Stdout.Flush()
	solve()
	Stdout.Flush()
}

// problem solver
func solve() {
	var a, b, c int
	a = getNextInt()
	b = getNextInt()
	c = getNextInt()
	var sum = a + b + c

	var inputs string = getNext()
	fmt.Fprintf(Stdout, "%d %s\n", sum, inputs)
}
